### ISSUE TITLE
@font-feature-values CSS at-rule incorrectly marked as an error.

### ISSUE THREAD

**Issue by Electricz0** [validator/validator#1683]
Code to reproduce the error:

```CSS
@font-feature-values Fira Code {
    @character-variant {
        alt-a: 1;
        alt-g: 2;
        alt-i-1: 3;
    }
}
```

Error message:

> Error Unrecognized at-rule `@font-features-values`
> From line 1, column 1; to line 1, column 20
> `@font-feature-values {↩	@c`

![image](https://github.com/validator/validator/assets/12022661/cdb38c39-5cbd-49e3-a056-207582dbba05)

**Comment by j9t** [comment]
It seems this feature is still [in draft stage](https://drafts.csswg.org/css-fonts/#font-feature-values), which could explain lack of support. @ylafon, are you the right contact to follow up with?

**Comment by sideshowbarker** [comment]
Please re-raise this issue against the CSS validator at https://github.com/w3c/css-validator/issues/new.

(For CSS checking, the HTML checker relies uses the CSS validator code as a library — so after any change for this gets made to the upstream CSS-validator code, we will just pull that change into the HTML checker.)

**Comment by Electricz0** [comment]
After ensuring it has not been done before, I have raised the issue against the CSS validator here:

https://github.com/w3c/css-validator/issues/475
